src/oscar/templates/oscar/partials/nav_primary.html

```diff
@@ -2,50 +2,62 @@
 {% load category_tags %}
 {% load i18n %}
 
-<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-    </button>
+<nav class="{% block nav_classes %}navbar navbar-expand-lg navbar-dark bg-dark{% endblock %}">
+    {% block navbar_toggler %}
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+    {% endblock %}
 
-    <a class="btn btn-secondary float-right btn-cart ml-auto d-inline d-sm-none" href="{% url 'basket:summary' %}">
-        <i class="fas fa-shopping-cart"></i>
-        {% trans "Basket" %}
-        {% if not request.basket.is_empty %}
-            {% if request.basket.is_tax_known %}
-                {% blocktrans with total=request.basket.total_incl_tax|currency:request.basket.currency %}
-                    Total: {{ total }}
-                {% endblocktrans %}
-            {% else %}
-                {% blocktrans with total=request.basket.total_excl_tax|currency:request.basket.currency %}
-                    Total: {{ total }}
-                {% endblocktrans %}
+    {% block navbar_cart %}
```

Comment:
Please change this to `navbar_basket`. Oscar uses "basket" everywhere instead of "cart", thanks to it's British origins.